tox.ini

```diff
@@ -253,7 +253,7 @@ commands =
     flaskcache{012,013}: nosetests {posargs} tests/contrib/flask_cache
     flask{010,011,012}: nosetests {posargs} tests/contrib/flask
     flask-autopatch{010,011,012}: ddtrace-run nosetests {posargs} tests/contrib/flask_autopatch
-    falcon{10,11,12}: nosetests {posargs} tests/contrib/falcon/test_middleware.py
+    falcon{10,11,12}: nosetests {posargs} tests/contrib/falcon/test_middleware.py tests/contrib/falcon/test_distributed_tracing.py
```

Comment:
Also not sure about this. It seemed silly to make a new test directive, but test_distributed doesn't actually depend on the method of patching either.

Reply:
can't you simply rewrite this directive in:
```
falcon{10,11,12}: nosetests {posargs} tests/contrib/falcon
```

Reply:
That tries to execute all the tests, including the autopatch tests (which it seems are usually kept separate by convention?) and misinterpreting the test mixing test_suite-py as actual tests to run as well.

I think if I rename test_suite.py and we're OK with doing the autopatch/middleware tests under the same directive, then it is possible to use `falcon{10,11,12}: nosetests {posargs} tests/contrib/falcon`

Reply:
sorry forgot that we have an autopatch test also for Falcon. It's fine to keep it separated like we do in other frameworks so all good here.